src/lib/structures/Command.ts

```diff
@@ -333,6 +334,20 @@ export type CommandOptionsRunType =
 	| 'GUILD_PRIVATE_THREAD'
 	| 'GUILD_ANY';
 
+/**
+ * The allowed values for {@link CommandOptions.runIn} as an enum.
+ * @since 2.0.0
+ */
+export const enum CommandOptionsRunTypeEnum {
+	DM = 'DM',
+	GUILD_TEXT = 'GUILD_TEXT',
+	GUILD_NEWS = 'GUILD_NEWS',
+	GUILD_NEWS_THREAD = 'GUILD_NEWS_THREAD',
+	GUILD_PUBLIC_THREAD = 'GUILD_PUBLIC_THREAD',
+	GUILD_PRIVATE_THREAD = 'GUILD_PRIVATE_THREAD',
+	GUILD_ANY = 'GUILD_ANY'
```

Comment:
I don't know if we should PascalCase the keys (like we do with every other enum or not)

Reply:
We should have them in PascalCase.

Reply:
Done